test/compound/TestSupplyVault.t.sol

```diff
@@ -578,19 +578,18 @@ contract TestSupplyVault is TestSetupVaults {
 
         vm.roll(block.number + 1000);
 
-        uint256 balance = daiSupplyVault.balanceOf(address(supplier1));
-        vm.prank(address(supplier1));
+        uint256 balance = daiSupplyVault.balanceOf(address(vaultSupplier1));
+        vm.prank(address(vaultSupplier1));
         daiSupplyVault.transfer(address(supplier2), balance);
 
-        uint256 expectedIndex = ERC20(comp).balanceOf(address(daiSupplyVault)).divWadDown(
-            daiSupplyVault.totalSupply()
-        );
+        uint256 rewardAmount = ERC20(comp).balanceOf(address(daiSupplyVault));
+        uint256 expectedIndex = rewardAmount.divWadDown(daiSupplyVault.totalSupply());
         uint256 rewardsIndex = daiSupplyVault.rewardsIndex();
         assertEq(expectedIndex, rewardsIndex);
 
-        (uint256 index1, uint256 unclaimed1) = daiSupplyVault.userRewards(address(supplier1));
+        (uint256 index1, uint256 unclaimed1) = daiSupplyVault.userRewards(address(vaultSupplier1));
         assertEq(index1, rewardsIndex);
-        assertGt(unclaimed1, 0);
+        assertApproxEqAbs(unclaimed1, rewardAmount, 1e6);
```

Comment:
Is it strictly greater than 0?

Reply:
I think this should be clearly tested since reward can be closed to 0 and thus the test would while the result is wrong

Reply:
This is tested now